settings.gradle.kts

```diff
@@ -1,7 +1,8 @@
-pluginManagement {
-  repositories {
-    maven(url = "https://nexus.ossim.io/repository/all-repos" )
-  }
-}
-
 rootProject.name = "omar-volume-cleanup"
+
+val downloadMavenUrl: String by settings
```

Comment:
Ahhh, that's what I couldn't figure out!

Reply:
Yeah I had to google it as well.

Although if you didn't want to use [property delegation](https://kotlinlang.org/docs/reference/delegated-properties.html#delegated-properties), I think you could also do something like:  `val downloadMavenUrl = settings["downloadMavenUrl"]`